CHANGELOG.md

```diff
@@ -2,6 +2,59 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Version 1.x reached End of Life on September 8th, 2021.** See the changelog entry [here](https://www.twilio.com/changelog/end-of-life-complete-for-unsupported-versions-of-the-programmable-video-sdk). Support for the 1.x version ended on December 4th, 2020.
 
+2.25.0 (in progress)
+====================
+
+New Features
+------------
+
+**Auto-switch default audio input devices**
+
+A LocalAudioTrack is said to be capturing audio from the default audio input device if:
+
+- it was created using the MediaTrackConstraints `{ audio: true }`, or
+- it was created using the MediaTrackConstraints `{ audio: { deviceId: 'foo' } }`, and "foo" is not available, or
+- it was created using the MediaTrackConstraints `{ audio: { deviceId: { ideal: 'foo' } } }` and "foo" is not available
+
+In previous versions of the SDK, if the default device changed (ex: a bluetooth headset is connected to a mac or windows laptop),
+the LocalAudioTrack continued to capture audio from the old default device (ex: the laptop microphone). Now, a LocalAudioTrack
+will switch automatically from the old default audio input device to the new default audio input device (ex: from the laptop microphone to the headset microphone).
+This feature is controlled by a new [CreateLocalAudioTrackOptions](https://sdk.twilio.com/js/video/releases/2.25.0/docs/global.html#CreateLocalAudioTrackOptions)
+property `defaultDeviceCaptureMode`, which can be set to `auto` or `manual`.
+
+The application can decide to capture audio from a specific audio input device by creating a LocalAudioTrack:
+
+- using the MediaTrackConstraints `{ audio: { deviceId: 'foo' } }`, and "foo" is available, or
+- using the MediaTrackConstraints `{ audio: { deviceId: { ideal: 'foo' } } }` and "foo" is available, or
+- using the MediaTrackConstraints `{ audio: { deviceId: { exact: 'foo' } } }` and "foo" is available 
+
+In this case, the LocalAudioTrack DOES NOT switch to another audio input device if the current audio input device is no longer available. (VIDEO-11701)
```

Comment:
I was a bit confused by this block - Is it explaining the old behavior? If so, I think the paragraph above is enough and we can get rid of this.

Reply:
@seancoleman2 This is different than setting the property to `manual` (old behavior). In this case, since the developer requested a SPECIFIC audio device (as opposed to whatever the default audio device is), the SDK will not switch devices in case the current device is no longer available, even if the property is set to `auto`.